src/api/app/views/webui/session/_form.html.haml

```diff
@@ -1,22 +1,20 @@
-%h3= pagetitle
+- text_color ||= ''
+- submit_position ||= ''
 
-= form_tag(form_url, options) do
-  - if proxy
+= form_tag(log_in_params[:url], log_in_params[:options]) do
+  - if proxy_mode?
     = hidden_field_tag(:context, 'default')
     = hidden_field_tag(:proxypath, 'reserve')
     = hidden_field_tag(:message, 'Please log in')
     = hidden_field_tag(:url, return_to_location)
   .form-group
-    = label_tag(:username, 'Username')
+    = label_tag(:username, 'Username', class: "#{text_color}")
     = text_field_tag(:username, nil, placeholder: 'User Name', required: true, id: 'user-login', class: 'form-control')
   .form-group
-    = label_tag(:password, 'Password')
+    = label_tag(:password, 'Password', class: "#{text_color}")
     = password_field_tag(:password, nil, placeholder: 'Password', required: true, id: 'user-password', class: 'form-control')
-  = submit_tag('Log In', name: 'login', class: 'btn btn-primary')
-  %span
-    - if proxy
-      or
-      = link_to('Register', CONFIG['proxy_auth_register_page'] + '?%22')
-    - elsif CONFIG['frontend_ldap_mode'] != :on
```

Comment:
`CONFIG['frontend_ldap_mode']` is still used? As a different propose than `CONFIG['ldap_on']`?

Reply:
interesting that in the documentation I just find mention to `ldap_mode` https://openbuildservice.org/help/manuals/obs-admin-guide/obs.cha.administration.html#_obs_proxy_mode_configuration